src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value.toLowerCase());
+  };
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    movie.title.toLocaleLowerCase().includes(query)
+    || movie.description.toLocaleLowerCase().includes(query)
+  ));
```

Comment:
still looks complicated, but okay
here you could have used destructuring, vars, new separate function like `containsQuery` – everything that could make it look simpler